### ISSUE TITLE
Loosing cursor on autosave

### ISSUE THREAD

**Issue by MartinMikusat** [MartinMikusat/dashboard#24]
After writing inside a note and autosaving, the text cursor is lost. This is caused by the "Note Editor" state being saved inside Firebase and then loaded from it. All the state is saved and loaded as strings so the cursor position is not being saved in any way.

**Proposed solution:**
Save and load the cursor position together with the "Note Editor" state.

**Comment by MartinMikusat** [comment]
No longer relevant after #30
